charts/nginx-proxy/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
Do we need to create a serviceaccount?

Reply:
It's by default in the chart generated by helm. As long as we keep the corresponding value to falsy it won't be created.

Reply:
I remember that, unless specifying a custom svcaccount, the svcaccount `default` is used on the same namespace.

Ideally, could we specify "no svcaccount" so that any request to the Kube API from this pod would be denied unless configured with `kubeconfig` ?

Reply:
I've set serviceAccount.create to false by default.
Do you prefer I remove the serviceaccount.yaml (and associated values) instead?

Reply:
Removed